README.md

```diff
@@ -61,8 +61,28 @@ ba = BitArray.new(16, ["0000111111110000"].pack('B*'), reverse_byte: false)
 ba.to_s # "0000111111110000"
```

Comment:
Remember! Until [this earlier PR](https://github.com/peterc/bitarray/pull/14) is merged in, this PR will appear larger than it really is.